### PR TITLE
Issue #194 - Diff for a specific file

### PR DESCRIPTION
**Pull request by fhavrlent**
Added diff for a specific file between commits or branches. Issue #193

### CODE REVIEW

**Comment by llrt** [comment]
Seems very good to me for a first PR. Nice job!

My comments;
- I still miss the general case, diffs betweens any two arbitrary commits. This may be achieved using another abbrev commit hash instead of `HEAD`
- If the compared file is the same, you may inform it only at the end of the command:
`git diff c9b4315 49e144d test.cson`

Using this syntax has the advantage that you may compare more than one file between commits using wildcards:
`git diff c9b4315 49e144d **/*.cson`

**Comment by fhavrlent** [comment]
Thank you 👏

- I thought that it was self explanatory, but I will add also that into PR.
- Might add this one to another section, about checking multiple specific files

File: README.md [hunk 1]
```diff
@@ -1333,6 +1334,21 @@ If you want to revert to changes made just 1 commit before c5f567, pass the comm
 (master)$ git checkout c5f567~1 -- file1/to/restore file2/to/restore
 ```
 
+<a href="i-want-to-list-changes-of-a-specific-file-between-commits-or-branches"></a>
```
Comment (RichardLitt) [inline README.md]: No more <a> tags needed.

**Comment by RichardLitt** [comment]
@fhavrlent Do you need help learning how to edit a commit and push?

**Comment by RichardLitt** [comment]
@fhavrlent Ping. :)

**Comment by RichardLitt** [comment]
Merged in 42472aaa940d05192dab1a12ff1421cc0ad80929. Thank you!

**Pull request closed without merging**